static/package.json

```diff
@@ -60,7 +60,7 @@
     "nodemon": "^2.0.22",
     "papaparse": "^5.3.2",
     "qs": "^6.11.0",
-    "query-string": "^8.1.0",
+    "query-string": "^7.1.3",
```

Comment:
Needed to downgrade to v7 because this package moved to Pure ESM in v8, which breaks our jest testing.